tol_lab_share/config/defaults.py

```diff
@@ -40,3 +40,6 @@
 
 # Options: 'json' or 'binary'
 SELECTED_ENCODER_FOR_FEEDBACK_MESSAGE = "binary"
+
+# Validate SSL certificate chain when connecting to https
+CERTIFICATES_VALIDATION_ENABLED = True
```

Comment:
If you want to include this new key, you also need to update https://github.com/sanger/tol-lab-share/blob/develop/tol_lab_share/types.py

Reply:
There's quite a few other keys missing in that `types.py` file as well.

Reply:
Ok, adding all missing

Reply:
Looks good.  I wonder if there's a linting rule we can turn on to check that instances of the type don't include values that aren't in the module type specification.  I've seen this a few times before.

Reply:
There is a --strict that includes as mandatory a lot of optional but I couldn't see for that flag in particular.